packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -106,6 +106,7 @@ export class OdspSummaryUploadManager {
                     headers: Object.keys(headers).length !== 0 ? true : undefined,
                     blobs,
                     size: postBody.length,
+                    sequenceNumber: referenceSequenceNumber,
```

Comment:
This should be called `referenceSequenceNumber`. `sequenceNumber` means something very different.